portal/i18next_gulpfile.js

```diff
@@ -8,33 +8,39 @@
  * run gulp --gulpfile i18next_gulpfile.js [task name]  will run individual task
  * NB:  should NOT run this in Production environment, the resulting modules in node_modules/ folder from running `npm install` should never be checked in
  */
-var gulp = require('gulp');
-var source = require('vinyl-source-stream');
-var request = require('request');
-var merge = require('merge2');
-var buffer = require('gulp-buffer');
-var del = require('del');
-var scanner = require('i18next-scanner');
-var concatPo = require('gulp-concat-po');
-var merge_json = require('gulp-merge-json');
-const path = require('path');
-const fs = require('fs');
-const i18nextConv = require('i18next-conv');
+var gulp = require("gulp");
+var source = require("vinyl-source-stream");
+var request = require("request");
+var merge = require("merge2");
+var buffer = require("gulp-buffer");
+var del = require("del");
+var scanner = require("i18next-scanner");
+var concatPo = require("gulp-concat-po");
+var merge_json = require("gulp-merge-json");
+var using = require("gulp-using");
+const path = require("path");
+const fs = require("fs");
+const i18nextConv = require("i18next-conv");
 /*
  * where the generated json/pot files from extraction of js files will reside
  */
-const translationSourceDir = path.join(__dirname, './translations/js/src/');
+const translationSourceDir = path.join(__dirname, "./translations/js/src/");
 /*
  * the path to the converted json file from po file of corresponding locale
  * JS files will consume the translated text from here
  * note json files are saved for each specific locale
  */
-const translationDestinationDir = path.join(__dirname,'./static/files/locales/');
+const translationDestinationDir = path.join(__dirname,"./static/files/locales/");
+
 /*
  * namespace
  */
-const nameSpace = 'frontend';
-const srcPotFileName = translationSourceDir+nameSpace+'.pot';
+const nameSpace = "frontend";
+const epromsNameSpace = "eproms";
+const truenthNameSpace = "truenth";
```

Comment:
Could you change this to "gil" to match Paul's directory names? Sorry for telling you "truenth" earlier!